lib/core/entity/customer.rb

```diff
@@ -33,7 +34,8 @@ def to_crm_hash
         GenderCode: { Value: GENDER_MAP[gender] },
         mas_AgeRange: { Value: AGE_RANGES_MAP[age_range] },
         BirthDate: iso8601_date_of_birth,
-        DoNotBulkEMail: !newsletter_subscription
+        DoNotBulkEMail: !newsletter_subscription,
+        Telephone2: contact_number
```

Comment:
Telephone2 is how it is named in CRM. That's the logical name that the CRM guys sent across. 
![screen shot 2015-08-03 at 09 54 55](https://cloud.githubusercontent.com/assets/591781/9034207/c2ee5f54-39c5-11e5-9ee4-2bfa854e7542.png)